css/properties/appearance.json

```
@@ -102,11 +102,16 @@
                 "prefix": "-webkit-"
               }
             ],
-            "opera_android": {
-              "version_added": "14",
-              "partial_implementation": true,
-              "prefix": "-webkit-"
-            },
+            "opera_android": [
+              {
+                "version_added": false
```

```suggestion
                "version_added": "60"
```

`npm run mirror` didn't do the right thing here because ab53ef31eedfb0e36af82ebfbe8190bc694d1cf5 landed on master after you branched off.